src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticAnalyzerExecutor.cs

```diff
@@ -113,7 +113,9 @@ public AnalyzerExecutor(AbstractHostDiagnosticUpdateSource hostDiagnosticUpdateS
                 // get compiler result builder map
                 var builderMap = analysisResult.ToResultBuilderMap(project, version, analyzerDriver.Compilation, analyzerDriver.Analyzers, cancellationToken);
 
+#pragma warning disable RS0012 // Do not call ToImmutableCollection on an ImmutableCollection value https://github.com/dotnet/roslyn-analyzers/issues/1430
```

Comment:
Might be worth more explicitly stating "disabled due to analyzer bug" so I don't have to pull up a web browser to figure out what that bug is.

Reply:
This is fixed in latest beta release, I will remove the suppression.